comeet/UI/MainMenu/MainMenuViewController.swift

```diff
@@ -73,9 +73,13 @@ class MainMenuViewController: BaseViewController {
     }
     
     func containerViewSwiped(gesture: UISwipeGestureRecognizer) {
+        let gesturePoint = gesture.location(in: containerView)
+        
         switch gesture.direction {
         case UISwipeGestureRecognizerDirection.left:
-            showAgenda()
+            if gesturePoint.y < containerView.frame.height - 80 { //bottom vie height
```

Comment:
Let's use the 80 magic number in a explicit constant. Or still better... get a reference/outlet to bottom view and use it's bounds.